packages/strapi/bin/strapi-start.js

```diff
@@ -11,13 +11,59 @@ const path = require('path');
 const cluster = require('cluster');
 
 // Public dependencies
-const fs = require('fs');
 const _ = require('lodash');
 const { cyan } = require('chalk');
+const chokidar = require('chokidar');
 
 // Logger.
 const { cli, logger } = require('strapi-utils');
 
+/**
+ * Init file watching to auto restart strapi app
+ * @param {Object} options - Options object
+ * @param {string} options.appPath - This is the path where the app is located, the watcher will watch the files under this folder
+ * @param {Strapi} options.strapi - Strapi instance
+ */
+const watchFileChanges = ({ appPath, strapi }) => {
+  const restart = () => {
+    if (strapi.reload.isWatching && !strapi.reload.isReloading) {
+      strapi.reload.isReloading = true;
+      strapi.reload();
+    }
+  };
+
+  const watcher = chokidar.watch(appPath, {
+    ignoreInitial: true,
+    ignored: [
+      /(^|[/\\])\../,
+      '**/tmp',
```

Comment:
It's `.tmp` or `tmp` ?

Reply:
That would likely depend on linux/windows. probably best to handle both cases for the future